content/en/os/1.14.x/concepts/bootstrap-containers/_index.markdown

```diff
@@ -0,0 +1,88 @@
++++
+title = "Bootstrap Containers"
+type = "docs"
+description = "Setting up the host with containers that start during boot." 
++++
+
+Bootstrap containers are a form of {{< ver-ref project="os" page="/concepts/components" >}}host container{{< /ver-ref >}} that you can use to run critical software before the node connects to an orchestrator.
+They give you subsantial power to configure and modify the system in ways that would otherwise be [difficult or impossible](#use-cases).
+
+## Lifecycle
+
+When you define a bootstrap container using user-data or the Bottlerocket API, the following sequence occurs on next boot:
+
+- [`systemd`](https://systemd.io/) starts all of the bootstrap containers concurrently,
+- the boot process pauses while the bootstrap containers start, run, and exit,
+- any bootstrap container configured to `{{< ver-ref project="os" page="/api/settings/bootstrap-containers#name_mode">}}mode=once{{< /ver-ref >}}` will change to `{{< ver-ref project="os" page="/api/settings/bootstrap-containers#name_mode">}}mode=off{{< /ver-ref >}}` once complete,
```

Comment:
Is this going to be rendered as?

```
any bootstrap container configured to `mode=once`
```

If so, I think this should say:

```
any bootstrap container configured with `mode=once`
```

And the same in other similar descriptions.

Reply:
The concern is `to` vs `with`?

(the `ver-ref` short code just deals with making a consistent link with a version)